From b07583769944dfcb77d584585f3c8bb7170f56ac Mon Sep 17 00:00:00 2001
From: John Audia <therealgraysky@proton.me>
Date: Thu, 25 Aug 2022 10:52:50 -0400
Subject: [PATCH] kernel: bump 5.10 to 5.10.138

All patches automatically rebased.

Signed-off-by: John Audia <therealgraysky@proton.me>
(cherry picked from commit fc89ec06928469021010c2f5776450f739af07a5)
---
 include/kernel-5.10                                       | 4 ++--
 ...ilter-nftables-update-table-flags-from-the-commi.patch | 8 ++++----
 .../generic/hack-5.10/321-powerpc_crtsavres_prereq.patch  | 2 +-
 .../octeon/patches-5.10/130-itus_shield_support.patch     | 2 +-
 ...add-quirk-for-Gateworks-PLX-PEX860x-switch-with-.patch | 2 +-
 5 files changed, 9 insertions(+), 9 deletions(-)

diff --git a/include/kernel-5.10 b/include/kernel-5.10
index 762e467468..4143799e89 100644
--- a/include/kernel-5.10
+++ b/include/kernel-5.10
@@ -1,2 +1,2 @@
-LINUX_VERSION-5.10 = .137
-LINUX_KERNEL_HASH-5.10.137 = c16bcd4d1453d7e645007d0a25270f52401331c7df81c364f57bde914e1616c8
+LINUX_VERSION-5.10 = .138
+LINUX_KERNEL_HASH-5.10.138 = 29a003bb8e0e3a45942f703370fb0b3460e6fdcbbad37424423c9cf831ab5ba8
diff --git a/target/linux/generic/backport-5.10/610-v5.13-10-netfilter-nftables-update-table-flags-from-the-commi.patch b/target/linux/generic/backport-5.10/610-v5.13-10-netfilter-nftables-update-table-flags-from-the-commi.patch
index 5ab2718ecb..eea0aed6aa 100644
--- a/target/linux/generic/backport-5.10/610-v5.13-10-netfilter-nftables-update-table-flags-from-the-commi.patch
+++ b/target/linux/generic/backport-5.10/610-v5.13-10-netfilter-nftables-update-table-flags-from-the-commi.patch
@@ -34,7 +34,7 @@ Signed-off-by: Pablo Neira Ayuso <pablo@netfilter.org>
  	struct nft_set			*set;
 --- a/net/netfilter/nf_tables_api.c
 +++ b/net/netfilter/nf_tables_api.c
-@@ -893,6 +893,12 @@ static void nf_tables_table_disable(stru
+@@ -917,6 +917,12 @@ static void nf_tables_table_disable(stru
  	nft_table_disable(net, table, 0);
  }
  
@@ -47,7 +47,7 @@ Signed-off-by: Pablo Neira Ayuso <pablo@netfilter.org>
  static int nf_tables_updtable(struct nft_ctx *ctx)
  {
  	struct nft_trans *trans;
-@@ -916,19 +922,17 @@ static int nf_tables_updtable(struct nft
+@@ -940,19 +946,17 @@ static int nf_tables_updtable(struct nft
  
  	if ((flags & NFT_TABLE_F_DORMANT) &&
  	    !(ctx->table->flags & NFT_TABLE_F_DORMANT)) {
@@ -70,7 +70,7 @@ Signed-off-by: Pablo Neira Ayuso <pablo@netfilter.org>
  	nft_trans_table_update(trans) = true;
  	list_add_tail(&trans->list, &ctx->net->nft.commit_list);
  	return 0;
-@@ -7931,11 +7935,10 @@ static int nf_tables_commit(struct net *
+@@ -7929,11 +7933,10 @@ static int nf_tables_commit(struct net *
  		switch (trans->msg_type) {
  		case NFT_MSG_NEWTABLE:
  			if (nft_trans_table_update(trans)) {
@@ -86,7 +86,7 @@ Signed-off-by: Pablo Neira Ayuso <pablo@netfilter.org>
  			} else {
  				nft_clear(net, trans->ctx.table);
  			}
-@@ -8148,11 +8151,9 @@ static int __nf_tables_abort(struct net
+@@ -8147,11 +8150,9 @@ static int __nf_tables_abort(struct net
  		switch (trans->msg_type) {
  		case NFT_MSG_NEWTABLE:
  			if (nft_trans_table_update(trans)) {
diff --git a/target/linux/generic/hack-5.10/321-powerpc_crtsavres_prereq.patch b/target/linux/generic/hack-5.10/321-powerpc_crtsavres_prereq.patch
index 4bb438295d..c4fd475b1f 100644
--- a/target/linux/generic/hack-5.10/321-powerpc_crtsavres_prereq.patch
+++ b/target/linux/generic/hack-5.10/321-powerpc_crtsavres_prereq.patch
@@ -16,7 +16,7 @@ Signed-off-by: Alexandros C. Couloumbis <alex@ozo.com>
 
 --- a/arch/powerpc/Makefile
 +++ b/arch/powerpc/Makefile
-@@ -61,19 +61,6 @@ machine-$(CONFIG_PPC64) += 64
+@@ -44,19 +44,6 @@ machine-$(CONFIG_PPC64) += 64
  machine-$(CONFIG_CPU_LITTLE_ENDIAN) += le
  UTS_MACHINE := $(subst $(space),,$(machine-y))
  
diff --git a/target/linux/octeon/patches-5.10/130-itus_shield_support.patch b/target/linux/octeon/patches-5.10/130-itus_shield_support.patch
index 15f514209d..a041331c20 100644
--- a/target/linux/octeon/patches-5.10/130-itus_shield_support.patch
+++ b/target/linux/octeon/patches-5.10/130-itus_shield_support.patch
@@ -20,7 +20,7 @@
  	}
 --- a/arch/mips/cavium-octeon/octeon-platform.c
 +++ b/arch/mips/cavium-octeon/octeon-platform.c
-@@ -773,7 +773,7 @@ int __init octeon_prune_device_tree(void
+@@ -774,7 +774,7 @@ int __init octeon_prune_device_tree(void
  	if (fdt_check_header(initial_boot_params))
  		panic("Corrupt Device Tree.");
  
diff --git a/target/linux/octeontx/patches-5.10/0004-PCI-add-quirk-for-Gateworks-PLX-PEX860x-switch-with-.patch b/target/linux/octeontx/patches-5.10/0004-PCI-add-quirk-for-Gateworks-PLX-PEX860x-switch-with-.patch
index 76e97ccbeb..28502d393f 100644
--- a/target/linux/octeontx/patches-5.10/0004-PCI-add-quirk-for-Gateworks-PLX-PEX860x-switch-with-.patch
+++ b/target/linux/octeontx/patches-5.10/0004-PCI-add-quirk-for-Gateworks-PLX-PEX860x-switch-with-.patch
@@ -22,7 +22,7 @@ Signed-off-by: Tim Harvey <tharvey@gateworks.com>
  #include <linux/platform_data/x86/apple.h>
  #include <linux/pm_runtime.h>
  #include <linux/suspend.h>
-@@ -5784,3 +5785,34 @@ static void nvidia_ion_ahci_fixup(struct
+@@ -5787,3 +5788,34 @@ static void nvidia_ion_ahci_fixup(struct
  	pdev->dev_flags |= PCI_DEV_FLAGS_HAS_MSI_MASKING;
  }
  DECLARE_PCI_FIXUP_FINAL(PCI_VENDOR_ID_NVIDIA, 0x0ab8, nvidia_ion_ahci_fixup);
-- 
2.34.1

